loomi/regulariza-morador-client
# Feature/auth integration back

## Feature
 - Login integration
---

**What I did:**

- Integrate login with backend
- Get Stamps and Nucleus and fill in the selects of LoginForm
 - create LoginContainer to store the logic behind these async requests
**How to test:**

1. In your browser go to [stage](http://project.netlify.app/) or [localhost](http://localhost:3000/)
2. Check if the component/view looks just like the [mockup](https://xd.adobe.com/) in different screen sizes (width between 1024px and 1920px)
3. Test the things that you think are worth testing, even the ones that are not in this description

![image](https://user-images.githubusercontent.com/32523401/134745368-f951db8e-9859-41aa-87ae-6accecc63dc7.png)
